assets/js/modules/analytics-4/index.js

```diff
@@ -169,19 +168,15 @@ export const registerWidgets = ( widgets ) => {
 			wrapWidget: false,
 			modules: [ 'analytics-4' ],
 			isActive: ( select ) => {
-				const isAnalyticsConnected =
-					select( CORE_MODULES ).isModuleConnected( 'analytics-4' );
-
-				/**
-				 * TODO: This widget should be shown only if the audience group
-				 * is set up for the current user. This should be fixed once
-				 * the audience settings become accessible without `analytics-4`
-				 * module being connected.
-				 * See: https://github.com/google/site-kit-wp/issues/8810 for
-				 * more details.
-				 */
```

Comment:
@kuasha420 @techanvil ok, we "disconnected" this CTA widget from the anlaytics-4 module, but how this is going to be registered if the analytics module is disconnected? If the Analytics module is disconnected, then it doesn't load its bundle, thus this code won't be executed, right?

PS: As far as I understand by "disconnected" you mean when the Analytics module is deactivated, right? Or you by that you mean when the Analytics module is active but not connected? Could you please clarify what you mean by that?

Reply:
Hi Eugene, I think the asset will still be loaded and the widgets registered if the module is not active or connected. The Asset will only not load if the module is disabled by the `googlesitekit_available_modules` filter. The module settings isn't available due to the `GETL:settings` route not being accessible for a deactivated module. 

Yes, by disconnected, I meant deactivated.

Reply:
Ah... okay, it is a bit confusing that deactivated module assets are loaded.

Reply:
Yeah, it would be nice if we could only load assets for modules which are active. It would be a good improvement to make in future, although it will need a bit of refactoring. It's evident that inactive module assets are still used for various connect CTAs (e.g. the existing `AdSenseConnectCTAWidget` as well as this one) and there could be some other use too that's less immediately evident...